frontend/src/app/app.module.ts

```diff
@@ -1,62 +1,62 @@
-import { UserProfileComponent } from './user-profile/user-profile.component';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 import { IMaskModule } from 'angular-imask';
 
+import { AppComponent } from 'app/app.component';
+import { AccountComponent} from 'app/components/account/account.component';
+import { ActivationComponent } from 'app/components/activation/activation.component';
+import { BannerComponent } from 'app/components/banner/banner.component';
+import { ContactComponent } from 'app/components/contact/contact.component';
+import { CookieLawBannerComponent } from 'app/components/cookie-law-banner/cookie-law-banner.component';
+import { FooterComponent } from 'app/components/footer/footer.component';
+import { FormErrorComponent } from 'app/components/form-error/form-error.component';
+import { HeaderComponent } from 'app/components/header/header.component';
+import { HomePageComponent } from 'app/components/homepage/homepage.component';
+import { HomepageOfferComponent } from 'app/components/homepage-offer/homepage-offer.component';
+import { LoginComponent } from 'app/components/login/login.component';
+import { MessagesComponent } from 'app/components/messages/messages.component';
+import { CreateOfferComponent } from 'app/components/offers/create-offer/create-offer.component';
+import { OfferDetailComponent } from 'app/components/offers/offer-detail/offer-detail.component';
+import { OrganizationComponent } from 'app/components/organization/organization.component';
+import { OrganizationContactComponent } from 'app/components/organization/organization-contact/organization-contact.component';
+import { OrganizationDetailsComponent } from 'app/components/organization/organization-details/organization-details.component';
+import { OrganizationOffersListComponent } from 'app/components/organization/organization-offers-list/organization-offers-list.component';
+import { OrganizationsComponent } from 'app/components/organizations/organizations.component';
+import { OrganizationCreateComponent } from 'app/components/organization/organization-create/organization-create.component';
+import { OrganizationsListComponent } from 'app/components/organizations/organizations-list/organizations-list.component';
+import { PasswordResetComponent } from 'app/components/password-reset/password-reset.component';
+import { PasswordResetConfirmComponent } from 'app/components/password-reset/password-reset-confirm.component';
+import { RedirectComponent } from 'app/components/redirect/redirect.component';
+import { RegisterComponent } from 'app/components/register/register.component';
+import { AboutUsComponent } from 'app/components/static/about-us.component';
+import { FaqOrganizationsComponent } from 'app/components/static/faq-organizations.component';
+import { FaqVolunteersComponent } from 'app/components/static/faq-volunteers.component';
+import { OfficeComponent } from 'app/components/static/office/office.component';
```

Comment:
Why the OfficeComponent has its own folder in static and other components haven't?

Reply:
So I think we can merge it like it is, but we agree that it is just a beginning of necessary changes.